galaxy.yml

```diff
@@ -19,5 +19,12 @@ tags:
 - cisco
 version: 1.3.1
 build_ignore:
-- tests/output/*
+- 'tests/output'
```

Comment:
Since ansible-2.10 this is always done by `build` https://github.com/ansible/ansible/blob/devel/lib/ansible/galaxy/collection.py#L922. The `build_ignore` key was also added for 2.10 so having this entry is redundant.

Reply:
Updated. Thank you. Any other feedback?